inputWidget/1.js

```diff
@@ -0,0 +1,22 @@
+const ele = document.querySelector('.inpElement');
+const button = document.querySelector('.numSubmit');
+const numbers = document.querySelector('.numbers');
+
+button.addEventListener('click', function() {
+    let num = Number(ele.value);
```

Comment:
use const since num is not being reassigned.
Add validation if the number is negative or more than 100